rasa/nlu/training_data/formats/markdown.py

```diff
@@ -74,6 +101,20 @@ def reads(self, s: Text, **kwargs: Any) -> "TrainingData":
             else:
                 self._parse_item(line)
                 self._load_files(line)
+
+        if self._deprecated_synonym_format_was_used:
+            raise_warning(
+                "You are using the deprecated training data format to declare synonyms."
+                " Please use the following format: \n"
+                "[<entity-text>]{'entity': '<entity-type>', 'value': "
```

Comment:
Single quotes would actually not work, right? I guess you have to escape the double quotes or use single quotes for the outer string